src/pages/Homepage/Services/Services.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import SystemTypography from '../../../components/Typography/Typography'
+import ApplicationCard from '../../../components/Card/Card'
+import { CardContent, CardMedia } from '@mui/material'
+import AppButton from '../../../components/Buttons/Button'
+import Box from '@mui/material/Box'
+import ServiceOutlineText from '../../../assets/images/outline-text/service.svg'
+import { ServicesData } from '../../../constants/Services/ServicesData'
+
+const Services = () => {
+
+  return (
+    <Box style={{display: 'flex', flexDirection: 'column', justifyContent: 'center', gap: '4.5rem'}}>
+        <Box style={{display: 'flex', flexDirection: 'column', alignItems: 'center', gap: '.5rem'}}>
+        <img src={ServiceOutlineText} alt='service outline'/>
```

Comment:
Transfer this whole code to your service_field.js then all our UI must fall in JSX

Reply:
Noted and Resolved.